imageprocessor/imageprocessor-plugin/src/main/java/pl/edu/uj/kimage/plugin/FlowStepFactory.java

```diff
@@ -0,0 +1,8 @@
+package pl.edu.uj.kimage.plugin;
+
+import pl.edu.uj.kimage.eventbus.EventBus;
+
+public interface FlowStepFactory<T extends FlowStep> {
+    T create();
+    FlowStepFactory withEventBus(EventBus eventBus);
```

Comment:
Still I don't know why you keep `EventBus` in `FlowStep`, naming or logic isn't quite good